src/app/pages/home/services/tasks.service.ts

```diff
@@ -96,7 +96,7 @@ export class TasksService {
             return (
               task.timestamp >= tenMinutesAgo &&
               task.timestamp < tenMinutesAgo + offsetForward &&
-              !task.completed
+              task.completed === false
```

Comment:
why was this changed ?

Reply:
My bad, this PR still had some old changes which replaced this. Changing it back.